### ISSUE TITLE
About Dataset Split

### ISSUE THREAD

**Issue by lutao2021** [wbhu/Tri-MipRF#27]
When I check your configuration file, I see that dataset.split is "trainval". So, is the number of metrics reported in your paper based on this config file?

**Comment by zf-666** [comment]
same question, psnr seems does't reach the paper if only use the train dataset.

**Comment by Fjzd** [comment]
same question

**Comment by wbhu** [comment]
> When I check your configuration file, I see that dataset.split is "trainval". So, is the number of metrics reported in your paper based on this config file?

Yes. This setting is borrowed from nerfacc's implementation of instant-ngp
